libafl_qemu/librasan/asan/src/lib.rs

```diff
@@ -32,13 +32,15 @@
 //!
 //! ## Features
```

Comment:
With `document-features` create, features are self-documenting (if they have comments in the toml), see:
https://github.com/AFLplusplus/LibAFL/blob/85bcafa8318b767d56dd807172354bf8cdb9252b/libafl/src/lib.rs#L6

Reply:
Oh cool. How's this?